src/users/User.java

```diff
@@ -5,6 +5,7 @@ public abstract class User {
     String name;
     String email;
 	String phoneNumber;
+	String department;
```

Comment:
department is not a variable in all subclasses

Reply:
Ok i got it